app/main.py

```diff
@@ -1 +1,43 @@
-#  write your code here
+class Animal:
+
+    def __init__(self, name: str, appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        return print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if not self.is_hungry:
+            return 0
+        else:
```

Comment:
You don't need to use an 'else' statement here. You can directly execute the code after the 'if' block if the condition is not met. This aligns with the checklist item: 'Don't use an extra else statement.'